src/components/pages/login.js

```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types';
 import LoginForm from '../login-form.js';
 import routes from '../../utils/routes.js';
 import { useTranslation } from 'react-i18next';
-import React, { useRef, useState } from 'react';
+import React, { Fragment, useRef, useState } from 'react';
```

Comment:
😱

Reply:
You can derictly derive my readiness for holidays by the number of times this happens :mx_claus:  Fixed, thanks!